src/main/java/forms/RegisterForm.java

```diff
@@ -1,4 +1,27 @@
 package forms;
 
-public class RegisterForm {
+public class RegisterForm extends Form{
+
+    private String username;
+    private String email;
+    private String password;
+    private String confirmpassword;
+
+    public RegisterForm(String title) {
+        super(title);
+    }
+
+    @Override
+    boolean validateForm() {
+        return false;
```

Comment:
Can you work on the validation. i.e. password and confirmpassword are the same, and that theyre 8+ characters and stuff like that.
Another thing would be email validation. i.e. contains the @

Reply:
agree needs validation. no need to go crazy with it tho, doesn't have to be flawless